lang/test/org/partiql/lang/eval/EvaluatingCompilerIntTest.kt

```diff
@@ -251,8 +251,9 @@ class EvaluatingCompilerIntTest : EvaluatorTestBase() {
         expectedPermissiveModeResult = "MISSING"
     )
 
-    @Test
-    fun castAlmostZeroDecimal() = runEvaluatorTestCase("cast(1e-2147483609 as int)", expectedResult = "0")
+    // TODO: see https://github.com/partiql/partiql-lang-kotlin/issues/784
+//    @Test
+//    fun castAlmostZeroDecimal() = runEvaluatorTestCase("cast(1e-2147483609 as int)", expectedResult = "0")
```

Comment:
A bug for `TypedOpBehavior.HONOR_PARAMETERS`. The fix is not covered in this PR.